src/Microsoft.AspNetCore.SignalR.Core/IClientProxy.cs

```diff
@@ -8,11 +8,11 @@ namespace Microsoft.AspNetCore.SignalR
     public interface IClientProxy
     {
         /// <summary>
-        /// Invokes a method on the connection(s) represented by the <see cref="IClientProxy"/> instance.
+        /// Fire-and-forget a method on the connection(s) represented by the <see cref="IClientProxy"/> instance.
```

Comment:
I do agree that this should be changed but this doesn't read well to me.

Reply:
I'd change it back. It does invoke a method. Just mention that it doesn't wait for the other side to send a reply.